tools/reviewer/job/sync/DiffConverter.java

```diff
@@ -80,14 +81,20 @@ public DiffConverter(GithubClient githubClient) {
 
     for (Thread thread : codeThreads) {
       for (Comment comment : thread.getCommentList()) {
+        // TODO: Store `githubCommentPosition` in the Diff.Thread in Firestore
+        int githubCommentPosition;
+        // `0` value means the comment isn't synced with GitHub
+        if (thread.getGithubCommentPosition() == 0) {
+          githubCommentPosition = getReviewCommentPositionFromGitHub(thread, githubPr);
```

Comment:
Please put the `TODO` inside `getReviewCommentPositionFromGitHub()` because on every call to that method, we'd like to store the result.

Reply:
Done.